src/patch/20.18.0/configure.py.patch

```diff
@@ -1,13 +1,14 @@
 --- a/configure.py
 +++ b/configure.py
-@@ -1598,7 +1598,9 @@ def configure_v8(o):
-   o['variables']['v8_use_siphash'] = 0 if options.without_siphash else 1
+@@ -1657,8 +1657,10 @@
    o['variables']['v8_enable_maglev'] = 1 if options.v8_enable_maglev else 0
    o['variables']['v8_enable_pointer_compression'] = 1 if options.enable_pointer_compression else 0
--  o['variables']['v8_enable_sandbox'] = 1 if options.enable_pointer_compression else 0
-+
-+  # o['variables']['v8_enable_sandbox'] = 1 if options.enable_pointer_compression else 0
-+
+   o['variables']['v8_enable_sandbox'] = 1 if options.enable_pointer_compression else 0
++  o['variables']['v8_enable_pointer_compression_shared_cage'] = 1 if options.enable_pointer_compression else 0
++  o['variables']['v8_enable_external_code_space'] = 1 if options.enable_pointer_compression else 0
    o['variables']['v8_enable_31bit_smis_on_64bit_arch'] = 1 if options.enable_pointer_compression else 0
-   o['variables']['v8_enable_shared_ro_heap'] = 0 if options.enable_pointer_compression or options.disable_shared_ro_heap else 1
+-  o['variables']['v8_enable_shared_ro_heap'] = 0 if options.enable_pointer_compression or options.disable_shared_ro_heap else 1
++  o['variables']['v8_enable_shared_ro_heap'] = 1 if options.enable_pointer_compression else 0
```

Comment:
I am not sure about this change at all just trying to reflect the changes applied here: https://github.com/avcribl/node/commit/2a185d128825d2b875da9d875d26fcf276ef04b4

Reply:
Let be borked for now